### PR TITLE
setting correct flag for brazil

### PR DESCRIPTION
**Pull request by Pedro-Souza**
## Proposed changes

setting correct flag for brazil

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ import store from '@/store'
 Vue.use(VueI18n)
 
 const messages = {
-  en: require('./en.json'),
-  nl: require('./nl.json'),
-  pt: require('./pt-br.json'),
-  pl: require('./pl.json')
+  'en': require('./en.json'),
+  'nl': require('./nl.json'),
```
Comment (faustbrian) [inline src/i18n/index.js]: No need to escape since it has no hyphens.

File: src/i18n/index.js [hunk 2]
```diff
@@ -5,10 +5,10 @@ import store from '@/store'
 Vue.use(VueI18n)
 
 const messages = {
-  en: require('./en.json'),
-  nl: require('./nl.json'),
-  pt: require('./pt-br.json'),
-  pl: require('./pl.json')
+  'en': require('./en.json'),
```
Comment (faustbrian) [inline src/i18n/index.js]: No need to escape since it has no hyphens.

File: src/i18n/index.js [hunk 3]
```diff
@@ -5,10 +5,10 @@ import store from '@/store'
...
+  'nl': require('./nl.json'),
+  'pt-br': require('./pt-br.json'),
+  'pl': require('./pl.json')
```
Comment (faustbrian) [inline src/i18n/index.js]: No need to escape since it has no hyphens.

**Comment by Pedro-Souza** [comment]
Done @faustbrian

**Comment by dated** [comment]
The failing test needs to be fixed here:

https://github.com/ArkEcosystem/explorer/blob/master/test/unit/specs/i18n/i18n.spec.js#L11

`expect(i18n.t('No Results', 'pt-br')).toBe('Sem resultados')`

**Comment by Pedro-Souza** [comment]
Done @dated @faustbrian

**Comment by dated** [comment]
I'll be looking for an icon set for the flags, after that this is ready to merge. 17.1 KB for a flag is simply too much.

**Review by dated — APPROVED** [review]

**Comment by faustbrian** [comment]
Mark this explicitly as a development command in the docs because this should not be used in production cronjobs. Ideally we would only mention it in the development docs, not the general cache docs.

**Pull request merged**